testutil/ciao-down/cloudinit.go

```diff
@@ -196,6 +196,9 @@ runcmd:
  - {{download . "https://download.fedoraproject.org/pub/fedora/linux/releases/24/CloudImages/x86_64/images/Fedora-Cloud-Base-24-1.2.x86_64.qcow2" (printf "/home/%%s/local/Fedora-Cloud-Base-24-1.2.x86_64.qcow2" .User)}}
  - {{template "CHECK" .}}
 
+ - curl -X PUT -d "Downloading xenial-server-cloudimg-amd64-disk1.img" 10.0.2.2:{{.HTTPServerPort}}
+ - {{download . "https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img" (printf "/home/%%s/local/xenial-server-cloudimg-amd64-disk1.img" .User)}}
+
```

Comment:
@markdryan there were some tests that didn't pass with the clear image, so we can definitely drop that - even though it doesn't take much upload time what's the point in having it if we don't use it? If we can get by with just the ubuntu cloudimg that is fine with me.